pkgs/applications/networking/cluster/sonobuoy/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  pname    = "sonobuoy";
+  version = "0.15.0";
+  # SHA of ${version} for the tool's help output
+  rev     = "7ad367535a6710802085d41e0dbb53df359b9882";
```

Comment:
Put it in a let statement outside the package. It doesn't need to be an attribute of the package.